src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar-content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 726px;
```

Comment:
You may try to create a variable or function for setting max-width.

Reply:
Don't think that's important here but made it for practice